Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM  mhart/alpine-node:11.0.0
```

Comment:
Why don't you use an official image like `FROM node:10-alpine`? 
(even version numbers are LTS by the way)

Reply:
You can use `node:10-alpine` instead

Reply:
Ok, I didn't find it during my research.
I will do it ASAP,

Thank's for the review.